typescript/appsync-graphql-dynamodb/index.ts

```diff
@@ -44,28 +44,38 @@ export class AppSyncCdkStack extends cdk.Stack {
       }`
     });
 
-    const itemsTable = new Table(this, 'ItemsTable', {
+    const itemsTable = new CfnTable(this, 'ItemsTable', {
       tableName: tableName,
-      partitionKey: {
-        name: `${tableName}Id`,
-        type: AttributeType.STRING
-      },
+      keySchema: [
+          {
+            attributeName: `${tableName}Id`,
+            keyType: "HASH",
+          }
+      ],
+      attributeDefinitions: [
+          {
+            attributeName: `${tableName}Id`,
+            attributeType: AttributeType.STRING,
+          }
+      ],
       billingMode: BillingMode.PAY_PER_REQUEST,
-      stream: StreamViewType.NEW_IMAGE
+      // streamSpecification: StreamViewType.NEW_IMAGE,
```

Comment:
I couldn't get this to resolve

Reply:
adding commented-out code is generally discouraged.

Reply:
will remove